pkg/gui.py

```
@@ -348,3 +332,89 @@ class SettingsMenu(OtherFrame):
+        tesseract_path = self.tesseract_path_entry.get()
+        debug_level = self.debug_level_var.get()
+
+        # Create a dictionary to hold the settings
+        settings = {
+            "tesseract_path": tesseract_path,
+            "debug_level": debug_level
+        }
+
+        # Load the settings from the JSON file
+        if os.path.exists("settings.json"):
+            with open("settings.json", "r") as settings_file:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)